src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -63,6 +63,8 @@ limitations under the License.
             <include cluster="General Commissioning" client="false" server="true" clientLocked="true" serverLocked="true">
                 <requireCommand>SetRegulatoryConfig</requireCommand>
             </include>
+            <include cluster="Joint Fabric Datastore" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Joint Fabric Administrator" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```

Comment:
What are the above two lines doing is to mandate the JF Administrator and JF Datastore Clusters in the Root Node Endpoint. This is incorrect as we have defined a new device type, the Joint Fabric Administrator Device Type (0x0130), which may sit on any endpoint (we even have a command for announcing the JF Endpoint to the server: AnnounceJointFabricAdministrator).

Reply:
I've updated the cluster definition to address this issue. Please review.